pkg/config/remote/client/client.go

```diff
@@ -15,6 +15,8 @@ import (
 	"sync"
 	"time"
 
+	"go.uber.org/atomic"
```

Comment:
Why uber's instead of https://pkg.go.dev/sync/atomic#Value ?

Reply:
https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/447936473#L452